modules/apps/notification/notification-api/src/main/java/com/liferay/notification/util/NotificationRecipientSettingUtil.java

```diff
@@ -40,6 +42,14 @@ public static Map<String, Object> toMap(
 				value = notificationRecipientSetting.getValueMap();
 			}
 
+			if (StringUtil.equals(
+					notificationRecipientSetting.getName(),
+					"singleRecipient")) {
+
+				value = GetterUtil.getBoolean(
+					notificationRecipientSetting.getValue());
+			}
```

Comment:
What do you think to sort and add else in this case? When the value can't be `xml` and `singleRecipient` ate the same time. Like this:

```java
if (StringUtil.equals(
		notificationRecipientSetting.getName(),
		"singleRecipient")) {

	value = GetterUtil.getBoolean(
		notificationRecipientSetting.getValue());
}
else if (Validator.isXml(notificationRecipientSetting.getValue())) {
	value = notificationRecipientSetting.getValueMap();
}
```

Reply:
makes sense